documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -294,18 +294,51 @@ livenessProbe:
 For more information about the `livenessProbe` and `readinessProbe` options, see xref:type-Probe-reference[Probe schema reference].
 
 [id='con-common-configuration-prometheus-{context}']
-=== `metrics`
+=== `metricsConfig`
 
-Use the `metrics` property to enable and configure Prometheus metrics.
+Use the `metricsConfig` property to enable and configure Prometheus metrics.
 
-The `metrics` property can also contain additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
+The `metricsConfig` property contains a reference to the ConfigMap containing additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
 Strimzi supports Prometheus metrics using Prometheus JMX exporter to convert the JMX metrics supported by Apache Kafka and ZooKeeper to Prometheus metrics.
 
-To enable Prometheus metrics export without any further configuration, you can set it to an empty object (`{}`).
+To enable Prometheus metrics export without further configuration, you can reference a ConfigMap containing empty data under a specified key.
+
+.Example ConfigMap with metrics configuration for Kafka
+
+[source,yaml,subs="+attributes"]
+----
+kind: ConfigMap
+apiVersion: {KafkaApiVersion}
+metadata:
+  name: my-configmap
+data:
+  my-key: '{"lowercaseOutputName":true,"rules":[{"labels":{"clientId":"$3","partition":"$5","topic":"$4"},"name":"kafka_server_$1_$2","pattern":"kafka.server<type=(.+),
```

Comment:
Similar comment as in the other PR ... this should be pretty printed. You should also try to create some shorter example (e.g. put some `// ...` in the middle etc.).